### ISSUE TITLE
Screen brightness isn't functional

### ISSUE THREAD

**Issue by AdrienKuhn** [danieleds/Asus-Zenbook-Ambient-Light-Sensor-Controller#16]
Hi,
I successfully compile als driver and als service for Asus Zenbook UX32A on Fedora 21.
It is functional for keyboard backlight, but not for screen brightness.

```
$ sudo cat /var/log/messages | grep als-controller
Mar  2 16:41:35 fc-zenbook als-controller[10061]: Started.
Mar  2 16:41:42 fc-zenbook als-controller[10061]: ALS enabled
Mar  2 16:41:42 fc-zenbook als-controller[10061]: Error opening /sys/class/backlight/acpi_video0/max_brightness
```

```
$ ls /sys/class/backlight/
intel_backlight
```

```
$ ls /sys/class/backlight/intel_backlight
actual_brightness
brightness
bl_power
device
max_brightness
power
subsystem
type
uevent
```

Distribution : Fedora 21
Kernel : 3.18.7-200.fc21.x86_64

Thank You

**Comment by AdrienKuhn** [comment]
If I replace lines containing "acpi_video0" by "intel_backlight" in main.cpp before compiling, screen brightness control is functional.

**Comment by danieleds** [comment]
Fixed in b26c929674eb1b1cbc73a414299348e9
